dev-tools/notice/overrides.json

```diff
@@ -19,3 +19,4 @@
 {"name": "github.com/JohnCGriffin/overflow", "licenceType": "MIT"}
 {"name": "github.com/elastic/ebpfevents", "licenceType": "Apache-2.0"}
 {"name": "go.opentelemetry.io/collector/config/configopaque", "licenceType": "Apache-2.0"}
+{"name": "github.com/elastic/go-quark", "licenceType": "Apache-2.0"}
```

Comment:
I think this is because we don't have the full license text in https://github.com/elastic/go-quark/blob/main/LICENSE.txt.  This was requested by legal, but I think we can discuss changing it to the full license, since it seem to cause both of these problems

Reply:
Please do ask about this because it seems like an uncommon pattern to follow.